.circleci/config.yml

```diff
@@ -64,12 +110,86 @@ jobs:
       - run:
           name: Invalidate CloudFront caches
           command: aws cloudfront create-invalidation --distribution-id EPTW7F3CB566V --paths "/*"
-          
+  
+
+  integration-tests-cocoapods:
+    macos:
+      xcode: "12.0"
+    working_directory: ~/purchases-ios/
+    shell: /bin/bash --login -o pipefail
+    steps:
+      - checkout
+      
+      - install-gems:
+          directory: "."
+      - install-gems:
+          directory: "IntegrationTests/CocoapodsIntegration"
+
+      - run:
+          name: Install Pods
+          command: |
+              cd IntegrationTests/CocoapodsIntegration/
+              pod install
```

Comment:
yeah, I thought about that, but when researching it I ran into this https://discuss.circleci.com/t/how-to-cache-cocoapods/28857/2. 
And it takes so little time that I figured it might not be worth it. 

Also, note that this particular `pod install` is only installing our local dependency, so it's pretty fast